### PR TITLE
Basic player and camera controller setup

### PR DESCRIPTION
**Pull request by mmont429**
Adds basic player controls [W, A, S, D, Space] are the inteded movement functions but the arrow keys also work
[GJ22-Development - DevPlayground - Windows, Mac, Linux - Unity 2021.3.2f1 Personal _DX11_ 2022-05-22 00-00-06.zip](https://github.com/Eacaw/GJ22-Development/files/8748472/GJ22-Development.-.DevPlayground.-.Windows.Mac.Linux.-.Unity.2021.3.2f1.Personal._DX11_.2022-05-22.00-00-06.zip)
.

### CODE REVIEW

File: Assets/Scripts/PlayerMovement.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+public class PlayerMovement : MonoBehaviour
+{
+    // Base user speed
+    [SerializeField] private float speed = 5.0f;
+    private Rigidbody rb;
+
+    public int MaxJump = 1;
+    int JumpCount = 0;
+
+    // Start is called before the first frame update
```
Comment (Eacaw) [inline Assets/Scripts/PlayerMovement.cs:14]: Can we get rid of these comments? We're going to see them a lot, no need to keep them in.

Comment (mmont429) [reply Assets/Scripts/PlayerMovement.cs:14]: Fine...

Comment (Eacaw) [reply Assets/Scripts/PlayerMovement.cs:14]: Cool, will be happy to approve, merge and close down this PR when you've done it.

Comment (mmont429) [reply Assets/Scripts/PlayerMovement.cs]: DONE

**Review by Eacaw — APPROVED** [review]
S'aight I suppose

**Pull request merged**